Cluster Programs

```diff
@@ -1,3 +1,68 @@
+Wednesday 07/25
+
+1) I have review Knitr and the reproducible data in coursera and given that it's implemented in Rstudio, 
+the process to use it should be pretty straight forward, the only problem is that we need to have the entire code running, 
+knitr will process the code and print the results.
+At the moment I haven't begin writing the movement to knitr because I'm still struggling with the optimization of the code.
+
+2) Documentation about the missing values in nclass was unavailable. 
+This variable comes in the original datasets from Harvard Dataverse, it is NOT created in the SAS code. 
+In my opinion, we could drop the NAs in the code, given that we are mainly grouping by nclass and NAs could mean that we are grouping 
+different things (e.g. paring nclass=4 with nclass=400) giving a wrong matching.
+
+3) In the original dataset from Harvard Dataverse you can find the variables Invnum and Invnum_N.
+Invnum: Initial inventor number: Patent + Invseq.
+Invnum_N: Disambiguated inventor number. This function assign a number to each of the individual inventors, 
+this way an inventor with more than one patent will have the same number. 
+In the program, we are using Invnum_N, which later it’s rename to Invnum.
+
+4) Regarding the next part, this is a good page for figuring out which distribution or method we are looking for
+https://berkorbay.github.io/fe522/02_Monte_Carlo_Simulation.html
+Looking the documentation, it said that before choosing a distribution for running the iterations, I need to perform and analysis
+on my dataset first. In this page there is a good starting point. 
+https://www.epa.gov/sites/production/files/2014-11/documents/montecar.pdf
+
+I need to figure this out before use the Monte Carlo function in R.
+This means that, I need to have my merge_patents dataset ready to begin this step. I'm trying to make it work with the small data sets. 
+
+There is a method to calculate the value of Pi using a Monte Carlo, it uses the radius of a circumference and then randomly put dots in
+a square of length 1, then it measure how many dots are inside the circumference and how many outside. 
+I think this could work in our program, if we take the radius as the 5, 10 or 20 miles of the different clusters. 
```

Comment:
I'm not sure that this exactly matches up to what we're doing. We'll be throwing down random SETS of points instead of random points...

It may end up being easier to just program some kind of loop to do our procedure 1000 times--I'm not sure., I can ask Jake for his input too if it looks very complicated to figure out,.